model/src/main/java/org/cbioportal/model/ClinicalAttribute.java

```diff
@@ -0,0 +1,61 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+
+public class ClinicalAttribute implements Serializable {
```

Comment:
I like lombok but every technology/library we introduce makes the project a little bit more complex, so I prefer keeping it simple/pure java for now, but yeah maybe later.